src/backend/app.js

```diff
@@ -21,15 +23,129 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+//----------------------------------------------------------------------------------------------------------------------
+// Respond with all meals in the future (relative to the when datetime) - /future-meals
+
+app.get('/future-meals', async (req, res) => {
```

Comment:
Since we have router defined, we shoudl have created these api's inside /api/meals.js and then should have used accessed api using router. We shall touch that during the class

Reply:
I understand now....
You mean 
mealsRouter.get('/future-meals', async (req, res) => {

Reply:
I tried using above 
Accessed API using 
http://localhost:5000/api/meals/future-meals